application/common/models/Password.php

```diff
@@ -185,9 +186,12 @@ public function save()
             /*
              * Throw exception based on exception type
              */
-            if ($e instanceof  PasswordReuseException) {
+            if ($e instanceof PasswordReuseException) {
                 \Yii::warning($log);
                 throw new ConflictHttpException(\Yii::t('app', 'Password.PasswordReuse'), 1469194882);
+            } elseif ($e instanceof PasswordStoreException) {
+                \Yii::warning($log);
+                throw new BadRequestHttpException($e->getMessage());
```

Comment:
Is a password store exception more likely to be due to a bad request or a server error? It seems like it could be either, but I don't know how to then decide which type of exception to throw here even if we wanted to take that into account.

Reply:
Good question. I would have to analyze all of the different passwordStore components' error possibilities. In the end, I don't think it matters because the UI handles all errors the same.

Reply:
Ahh, okay.